src/apps/capitals.jsx

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import capitalsData from "./capitalsData.json";
+
+const Territory = styled.div`
+  text-transform: capitalize;
+`;
+
+const Capital = styled.div`
+  text-transform: capitalize;
+  font-weight: bold;
+`;
+
+//=========================================
+
+function Renderer({ data }) {
+  return (
+    <div>
+      Capital of <Territory>{data.territory}</Territory> is{" "}
+      <Capital>{data.capital}</Capital>
+    </div>
```

Comment:
Divs are causing unnecessary line breaks 
<img width="1051" alt="Screenshot 2021-09-16 at 1 26 51 PM" src="https://user-images.githubusercontent.com/8065913/133604190-710c66e6-09a7-43b2-aabe-b713196cb386.png">

<img width="908" alt="Screenshot 2021-09-16 at 1 27 10 PM" src="https://user-images.githubusercontent.com/8065913/133604186-02602da6-4f46-4b70-a3c3-b144b54a2f6d.png">

Reply:
Fixed, thanks!

Actually this was in my local changes, missed pushing it 😅